source/blog/2017-07-31-fedora-atomic-25-to-26-upgrade.md

```diff
@@ -0,0 +1,260 @@
+
+Receiving delta parts: 0/27 5.3 MB/s 26.7 MB/355.4 MB
+Signature made Sun 23 Jul 2017 03:13:09 AM UTC using RSA key ID 812A6B4B64DAB85D
+  Good signature from "Fedora 26 Primary <fedora-26-primary@fedoraproject.org>"
+
+27 delta parts, 9 loose fetched; 347079 KiB transferred in 105 seconds                                                                                                                                            
+Copying /etc changes: 22 modified, 0 removed, 58 added
+Transaction complete; bootconfig swap: yes deployment count change: 1
+Upgraded:
+  GeoIP 1.6.11-1.fc25 -> 1.6.11-1.fc26
+  GeoIP-GeoLite-data 2017.04-1.fc25 -> 2017.06-1.fc26
+  NetworkManager 1:1.4.4-5.fc25 -> 1:1.8.2-1.fc26
+  ...
+  ...
+  setools-python-4.1.0-3.fc26.x86_64
+  setools-python3-4.1.0-3.fc26.x86_64
+Run "systemctl reboot" to start a reboot
+[vagrant@host ~]$ sudo reboot
+Connection to 192.168.121.217 closed by remote host.
+Connection to 192.168.121.217 closed.
+```
+
+After reboot we can log in and see the status:
+
+```nohighlight
+$ vagrant ssh
+[vagrant@host ~]$ rpm-ostree status
+State: idle
+Deployments:
+● fedora-atomic-26:fedora/26/x86_64/atomic-host
+                Version: 26.91 (2017-07-23 03:12:08)
+                 Commit: 0715ce81064c30d34ed52ef811a3ad5e5d6a34da980bf35b19312489b32d9b83
+           GPGSignature: 1 signature
+                         Signature made Sun 23 Jul 2017 03:13:09 AM UTC using RSA key ID 812A6B4B64DAB85D
+                         Good signature from "Fedora 26 Primary <fedora-26-primary@fedoraproject.org>"
+
+  fedora-atomic:fedora-atomic/25/x86_64/docker-host
+                Version: 25.154 (2017-07-04 01:38:10)
+                 Commit: ce555fa89da934e6eef23764fb40e8333234b8b60b6f688222247c958e5ebd5b
+[vagrant@host ~]$ cat /etc/fedora-release 
+Fedora release 26 (Twenty Six)
+```
+
+We are now on Fedora 26 Atomic Host. Now is a good time to check your
+services (most likely running in containers) to see if they are still
+working. If not, then you always have the rollback command: `sudo
+rpm-ostree rollback`.
+
+**NOTE:** Over time you can see updated commands for upgrading Atomic
+          Host between releases by visiting [this](https://fedoraproject.org/wiki/Atomic_Host_upgrade)
+          wiki page.
+
+# Appendix A: Upgrading Systems with Kubernetes
+
+Fedora 25 Atomic Host ships with Kubernetes v1.5.3, and Fedora 26
+Atomic Host ships with Kubernetes v1.6.7. **Before** upgrading systems
+participating in an existing Kubernetes cluster from 25 to 26, there
+are a few configuration changes to make.
+
+## node servers
+
+If your `KUBELET_ARGS` in `/etc/kubernetes/kubelet` point to the
+manifests directory using the argument `--config`, you need to change
+the argument name to `--pod-manifest-path`. In Kubernetes 1.6, the
+`--config` argument is no longer valid. Also in `KUBELET_ARGS`, you 
+need to add the argument `--cgroup-driver=systemd`.
+
+Need example here:
```

Comment:
There may be other options in there, depending on how they've set it up. I don't want to tell people, remove everything and replace it w/ XXXX, if I don't know what all is in there.